js/toolbar/annotation-tool.js

```diff
@@ -22,6 +23,10 @@ function AnnotationTool(parent){
     // Click to change annotation button
     this.onClickChangeBtn = function(){
         var btnType = this.getAttribute("data-type") || "";
+        var btnSubtype = null;
+        if(btnType === "ARROWLINE"){
+            btnSubtype = this.getAttribute("data-arrow-type")
```

Comment:
Given that we're going to have a different UI for toolbar, just remove any notion of subtype from here. Besides it's confusing that in the SVG definition you've called this `data-subtype` and in here you're doing `data-arrow-type`.

So like I said, just remove the notion that there are different types of arrowheads from the toolbar.